department-of-veterans-affairs/caseflow
# Make ColocatedActionsDropdown redirect to URL onChange

Connects #6851.

First step in unifying the task action dropdowns for the various roles.

### Test plan
1. Log in as a co-located admin staffer
2. Select a case
3. Select the "Send back to attorney" option from the dropdown
4. Notice how it works! Huzzah!
